hack/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM gcr.io/cf-build-service-public/ci/kpack-builder
+USER root
+RUN chmod -R 755 /cnb/lifecycle
+USER 1000:1000
```

Comment:
I had to do this since the latest builder seems incorrect. The /cnb/lifecycle directory has incorrect permissions that prevents pack from running the appropriate lifecycle binary once it drops permissions (even in trusted workflows during export).

Reply:
This can be removed once the bug in the kpack builder is fixed.